package.json

```diff
@@ -77,5 +77,8 @@
     "type": "opencollective",
     "url": "https://opencollective.com/accounts-js",
     "logo": "https://opencollective.com/accounts-js/logo.txt"
+  },
```

Comment:
can you please delete changeset here? The cli is already set in the dev deps :)

Reply:
Okay, I think that's done.